cocos/ui/UIEditBox/UIEditBoxImpl.h

```diff
@@ -54,13 +54,28 @@ namespace cocos2d {
             virtual void setInputFlag(EditBox::InputFlag inputFlag) = 0;
             virtual void setMaxLength(int maxLength) = 0;
             virtual int  getMaxLength() = 0;
-            virtual void setTextHorizontalAlignment(cocos2d::TextHAlignment alignment) = 0;
+            virtual void setTextHorizontalAlignment(TextHAlignment alignment) = 0;
             virtual void setReturnType(EditBox::KeyboardReturnType returnType) = 0;
             virtual bool isEditing() = 0;
             
             virtual void setText(const char* pText) = 0;
             virtual const char* getText(void) = 0;
             virtual void setPlaceHolder(const char* pText) = 0;
+            virtual const char* getPlaceHolder(void) = 0;
+
+            virtual const char* getFontName() { return ""; }
```

Comment:
I think all these method should be virtual method and needed be implemented in the platform-impl class.  Eg, in Tizen platform, the `getInputMode` method might be return the wrong input mode.  Because in UIEditboxImpl-tizen.h, the `getInputMode1 method is not properly implemented.